src/WpfMath.Tests/ApprovalTestUtils.fs

```diff
@@ -0,0 +1,57 @@
+module WpfMath.Tests.ApprovalTestUtils
+
+open System.Reflection
+
+open ApprovalTests
+open ApprovalTests.Namers
+open ApprovalTests.Reporters
+open Newtonsoft.Json
+open Newtonsoft.Json.Converters
+open Newtonsoft.Json.Serialization
+
+open System.Text
+open WpfMath
+
+[<assembly: UseReporter(typeof<DiffReporter>)>]
+[<assembly: UseApprovalSubdirectory("TestResults")>]
+do ()
+
+type private InnerPropertyContractResolver() =
+    inherit DefaultContractResolver()
+    member private __.DoCreateProperty(p, ms) =
+        let prop = base.CreateProperty(p, ms)
+        prop.Readable <- true
```

Comment:
IIRC you can init properties in ctor.

```fsharp
let prop = base.CreateProperty(p, ms, Readable = true)
```

Reply:
Wait, really? It works with functions and not only with real "constructors"? Wow.

Reply:
@ForNeVeR Oh, not works. :( My bad, need to sleep more.

Reply:
Wait, it works actually